docs/deploy_streamlit_app.md

```diff
@@ -14,8 +14,8 @@ Once you have your invite you're ready to deploy! It's really straightforward, j
 
 1. Add your Streamlit app to a public GitHub repo
 2. Add a requirements file to manage any external dependencies
-    1. [Python dependencies](/deploy_streamlit_app.html#python-dependencies)
-    2. [apt-get dependencies](/deploy_streamlit_app.html#apt-get-dependencies) (for Linux applications outside python environment)
+   1. [Python dependencies](/deploy_streamlit_app.html#python-dependencies)
+   2. [apt-get dependencies](/deploy_streamlit_app.html#apt-get-dependencies) (for Linux applications outside python environment)
```

Comment:
Looks like these changes are autoformatting when I merged in `develop`. I tried removing the autoformat in my editor, but I think it's in the git precommit update

Reply:
That format is fine...the formatting on `latest` appears to be incorrect in the same way. I think those are actually supposed to be `-`, but I can fix that in a separate PR